app/views/user_api_keys/new.html.erb

```diff
@@ -28,20 +28,8 @@
   <%= hidden_field_tag 'push_url', @push_url %>
   <%= hidden_field_tag 'public_key', @public_key%>
   <%= hidden_field_tag 'scopes', @scopes%>
-  <%= submit_tag t('user_api_key.authorize'), class: 'btn btn-danger', id: 'submit' %>
+  <%= submit_tag t('user_api_key.authorize'), class: 'btn btn-danger' %>
 <% end %>
-<script>
-  window.__submitted = false;
-
-  // prevent double submission which would invalidate the nonce
-  document.getElementById('submit').addEventListener('click', function(e){
-    if (window.__submitted) {
-      e.preventDefault();
-    } else {
-      window.__submitted = true;
-    }
-  });
-</script>
```

Comment:
These are simply removed, because the intended behavior is already the default for [`submit_tag` `disable_with` helper](https://api.rubyonrails.org/v5.0/classes/ActionView/Helpers/FormTagHelper.html#method-i-submit_tag).